qutebrowser/utils/usertypes.py

```diff
@@ -246,6 +247,52 @@ def reset(self):
                      'err_config', 'err_key_config'], is_int=True, start=0)
 
 
+# Where a download should be saved
+class DownloadTarget:
```

Comment:
While I see your goal there, it seems quite weird - do you have any alternative ideas on how to do this kind of thing? I'll think about it some more later, after reviewing the other PRs :wink: